.github/workflows/php.yml

```diff
@@ -1,119 +1,121 @@
-        runs-on: ubuntu-latest
-        steps:
-            - name: Setup PHP
-              uses: shivammathur/setup-php@v2
-              with:
-                php-version: '7.4'
-
-            - name: Checkout
-              uses: actions/checkout@v2.0.0
-
-            - name: Cache dependencies
-              uses: actions/cache@v2
-              with:
-                path: vendor
-                key: php-${{ hashFiles('composer.lock') }}
-
-            - name: Install dependencies
-              run: composer install
-
-            - name: Run PHP-CS-Fixer
-              run: ./vendor/bin/php-cs-fixer fix --dry-run --diff --using-cache=no
-
-    # Run PHPStan against the module and a PrestaShop release
-    phpstan:
-        name: PHPStan
-        runs-on: ubuntu-latest
-        strategy:
-            matrix:
-                presta-versions: ['nightly']
-        steps:
-            - name: Setup PHP
-              uses: shivammathur/setup-php@v2
-              with:
-                php-version: '7.4'
-
-            - name: Checkout
-              uses: actions/checkout@v2.0.0
-
-            # Add vendor folder in cache to make next builds faster
-            - name: Cache vendor folder
-              uses: actions/cache@v1
-              with:
-                path: vendor
-                key: php-${{ hashFiles('composer.lock') }}
-
-            # Add composer local folder in cache to make next builds faster
-            - name: Cache composer folder
-              uses: actions/cache@v1
-              with:
-                path: ~/.composer/cache
-                key: php-composer-cache
-
-            - run: composer install
-
-            # Docker images prestashop/prestashop may be used, even if the shop remains uninstalled
-            - name: Execute PHPStan on PrestaShop (Tag ${{ matrix.presta-versions }})
-              run: ./tests/phpstan.sh ${{ matrix.presta-versions }}
-
-    phpunit:
-      name: PHPUnit
-      runs-on: ubuntu-latest
-
-      steps:
-        - name: Setup PHP
-          uses: shivammathur/setup-php@v2
-          with:
-            php-version: '7.4'
-
-        - name: Checkout
-          uses: actions/checkout@v2.0.0
-
-        # Add vendor folder in cache to make next builds faster
-        - name: Cache vendor folder
-          uses: actions/cache@v1
-          with:
-            path: vendor
-            key: php-${{ hashFiles('composer.lock') }}
-
-        # Add composer local folder in cache to make next builds faster
-        - name: Cache composer folder
-          uses: actions/cache@v1
-          with:
-            path: ~/.composer/cache
-            key: php-composer-cache
-
-        - run: composer install
-
-        - name: PHPUnit
-          run: ./vendor/bin/phpunit -c ./tests/phpunit.xml
+  # Check there is no syntax errors in the project
+  php-linter:
+    name: PHP Syntax check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
:warning: ps_distributionapiclient must PS 8.2 & PS 9.0, so PHP 7.2 to PHP 8.4

Reply:
@Progi1984 it's ok for review now